### DISCUSSION TITLE
[Feature request] Icon seperate height and width settings in ini

### DISCUSSION THREAD

**Discussion by MrMainn** [complexlogic/flex-launcher#48] [Ideas]
Hello,

First, I would like to say that your app is great, it is simple, streamlined and exactlty what I was looking for for my HTPC build. 

After reading the ini configuration page, I see that we can only set-up the icon size in one dimension, which is the side of a square. 

I am using poster shaped Icons, so far, I created a transparent square and put the poster in the middle of the square but I am loosing a lot of space in between icons.

Would it be possible to have the option to chose the height and width of the icons instead of just one side ?

**Comment by complexlogic** [comment]
That's an interesting idea. I designed the launcher to use square icons, though. Have you tried reducing the `IconSpacing` setting?

**Reply by MrMainn** [reply]
Yes, I basically set 0 but I am loosing some real estate, see below

![image](https://github.com/complexlogic/flex-launcher/assets/152399254/e18dd76b-e51b-4710-983a-30c653ecd8e2)

**Reply by MrMainn** [reply]
If we could optimize height and width I could make these posters bigger.

**Comment by MrMainn** [comment]
Additional question, how is the iconsize limited ?

When I go a bit over 1000 pixels, te icone shrink back.

**Reply by complexlogic** [reply]
It's currently limited to a maximum of 1024 pixels. It's defined [here](https://github.com/complexlogic/flex-launcher/blob/master/src/launcher.h#L18), if you compile yourself you can change it.
